uw_sws/enrollment.py

```diff
@@ -49,11 +50,15 @@ def get_enrollment_by_regid_and_term(regid, term):
                                 term.year,
                                 term.quarter,
                                 regid)
-    return _json_to_enrollment(get_resource(url))
+    return _json_to_enrollment(get_resource(url), url)
```

Comment:
get_resource raises a datafailureexception if the status isn't 200 - does the production sws return a proper 404 status code?  If so, our 404 resources should have a .http-headers file, and restclients will use that to load various parts of the response object.  here's an example: https://github.com/uw-it-aca/uw-restclients/blob/master/restclients/resources/bridge/file/api/admin/users/195.DELETE.http-headers